common4j/src/main/com/microsoft/identity/common/java/util/ported/Pair.java

```diff
@@ -23,10 +23,54 @@
 
 package com.microsoft.identity.common.java.util.ported;
 
+import java.util.List;
+import java.util.Objects;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import lombok.AllArgsConstructor;
+import lombok.NonNull;
 
 @AllArgsConstructor
 public class Pair<T, U> {
```

Comment:
Do you just want to take the code directly from AOSP: https://github.com/aosp-mirror/platform_frameworks_base/blob/master/core/java/android/util/Pair.java

Reply:
Decided not to do it for now.